app/screens/move-money-screen/money-money-screen.tsx

```diff
@@ -173,6 +173,12 @@ export const MoveMoneyScreenDataInjected = ({ navigation }) => {
 export const MoveMoneyScreen = (
   ({ walletIsActive, navigation, loading, error, transactions,
     refetch, amount, amountOtherCurrency, isUpdateAvailable }) => {
+  
+  useEffect(() => {
+    StatusBar.setTranslucent(false)
```

Comment:
if this supposed to be set for the whole app as a permanent settings?
In this case we could set that earlier than `money-money-screen.tsx`

I remember trying to play with the StatusBar bar some time ago. I think there are some bug when going to the quiz tab and coming back to the main page. 

I think a useful issue would be a large refactoring the all the style/css. right now there are a lot of quick fixes on the style without a coherent structure. this will get more important as other project wants to fork this project to have their own wallet.

I've created a ticket for that: https://github.com/GaloyMoney/galoy-mobile/issues/59

Reply:
The "StatusBar.setTranslucent(false)" line was unneeded, so I deleted it. I had copied all three lines together from another screen.